common/models/transducer/alignment.py

```diff
@@ -11,7 +11,7 @@ def rna_alignment(source, **kwargs):
   # targets: (B, U-1)
   # input_lengths (B,)
   # label_lengths (B,)
-  from .rna_pure_tf import tf_forward_shifted_rna
+  from .rna_pure_tf import rna_align_sum_max_pure_tf
```

Comment:
You think this is a better name?
(I never was talking about the function name.)